module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,60 @@
+#include "compression.hpp"
+
+#include <algorithm>
+#include <iostream>
+#include <utility>
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(
```

Comment:
You can make some type alias for these long & complex types.

Reply:
Thanks, done :)